sourcegraph/about-replatform
# Add podcast records

Closes #42 by adding building/ parsing for individual podcast records. Also updates records to work within our new MD/ MDX build processes.

## To Test
- Navigate to any podcast record and ensure that it matches what's on prod.
- Test static builds to ensure that all MD passes.
